lib/kredits.js

```diff
@@ -22,16 +22,16 @@ function capitalize(word) {
 
 class Kredits {
 
-  constructor(provider, signer, addresses) {
+  constructor(provider, signer, options = {}) {
     this.provider = provider;
     this.signer = signer;
 
```

Comment:
What about a destructuring of the options to make it more easy to see the used options?

Reply:
something like this: 
` let { address, abis, ipfsConfig } = options ` 
is that correct?

Reply:
Jup